examples/externalProfileDemo.html

```diff
@@ -0,0 +1,337 @@
+                                for (let i = 0; i < entities.length; i++) {
+                                    document.getElementById('tableBody').insertAdjacentHTML('beforeend',
+                                        `<tr>
+                                            <td>
+                                                <a id="entity-${i}">${entities[i].lastName}, ${entities[i].firstName}</a>
+                                            </td>
+                                            <td>
+                                                <a id="org-${i}">${entities[i].externalOrganization.id}</a>
+                                            </td>
+                                        </tr>`
+                                    );
+                                    // Attach listeners to entities to call sdk methods
+                                    document.getElementById(`entity-${i}`).addEventListener('click', function() {
+                                        clientApp.contacts.showExternalContactProfile(entities[i].id);
+                                    });
+                                    document.getElementById(`org-${i}`).addEventListener('click', function() {
+                                        clientApp.contacts.showExternalOrganizationProfile(entities[i].externalOrganization.id);
+                                    });
+
+                                }
+                            });
+                            // Hide progress bar after auth is done
+                            let authenticatingEl = document.querySelector('.authenticating');
+                            authenticatingEl.classList.add('hidden');
+                        } else {
+                            setErrorState('You do not have the proper permissions to view external contacts.');
+                        }
+                    })
+                    .catch(function (err) {
+                        if (!authenticated) {
+                            setErrorState('Failed to Authenticate with PureCloud');
+                        } else if (!userDataAcquired) {
+                            setErrorState('Failed to locate user in PureCloud');
+                        }
+                    })
+
+                function setErrorState(errorMsg) {
+                    // Display error text in progress bar
+                    document.querySelector('.progress-label').innerHTML = errorMsg;
+                    document.querySelector('.progress-bar').className += ' progress-bar-danger';
+                    updateProgressBar(100);
+                }
+
+                /**
+                 * Parse the permissions array and check whether or not the match the specified required ones.
+                 *
+                 * @returns A boolean indicating if the user possesses the required permissions.
+                 */
+                function checkPermission(permissions, permissionValue) {
```

Comment:
I need to see if my new build process will allow for us to pull stuff like this and query string parsing out into a utils folder at this same level.  No need to do it for this PR, but it just reminded me.